subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformValuesInjectionIntegrationTest.groovy

```diff
@@ -113,15 +117,15 @@ class ArtifactTransformValuesInjectionIntegrationTest extends AbstractDependency
                 void setMissingInput(String missing)
             }
             
-            abstract class MakeGreenAction extends ArtifactTransform {
+            abstract class MakeGreenAction implements ArtifactTransformAction {
                 @TransformParameters
```

Comment:
I think mixing `Transform` and `ArtifactTransform` as a common prefix for stuff related to transforms is somewhat confusing. Here we have `@TransformParameters` but `ArtifactTransformAction` etc. Then again, `@PrimaryInput` and `@PrimaryInputDependencies` doesn't have anything apart from the package name to identify them as belonging to transforms.

We should address this in a separate PR, just wanted to mention it here, too.